### PR TITLE
Adding namespace to helm and kubectl commands

### PR DESCRIPTION
**Pull request by shigupt202**
**Task name**: HelmDeployV0

**Description**: Adding namespace to `helm get manifest` and `kubectl get pods -o json` commands

**Documentation changes required:** N

**Added unit tests:**  Y

**Attached related issue:**  https://github.com/microsoft/azure-pipelines-tasks/issues/13197

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/HelmDeployV0/src/utils.ts [hunk 1]
```diff
@@ -70,6 +71,8 @@ export function getManifestsFromRelease(helmCli: helmcli, releaseName: string):
     helmCli.setCommand('get');
     helmCli.addArgument('manifest');
     helmCli.addArgument(releaseName);
+    if (namespace)
+        helmCli.addArgument('--namespace '.concat(namespace));
```
Comment (ds-ms) [inline Tasks/HelmDeployV0/src/utils.ts:75]: Add each argument separately? Doesn't it convert to `"--namespace my-namespace"` instead?

File: Tasks/HelmDeployV0/Tests/TestSetup.ts [hunk 2]
```diff
@@ -277,17 +277,34 @@ else {
...
-}
+if (process.env[shared.TestEnvVars.namespace]) {
```
Comment (ds-ms) [inline Tasks/HelmDeployV0/Tests/TestSetup.ts:280]: Add a test case for the same.. where you set a different namespace

Comment (shigupt202) [reply Tasks/HelmDeployV0/Tests/TestSetup.ts:280]: There's already a test case where a different namespace is set.

File: Tasks/HelmDeployV0/src/utils.ts (hunk 1 above)
Comment (shigupt202) [reply Tasks/HelmDeployV0/src/utils.ts:75]: Yes, but that's what it should be right? This is how it's done in `helm install` and `helm upgrade` commands. Wanted to keep it consistent.

**Pull request merged**